pkg/sdk/grants.go

```diff
@@ -250,3 +252,32 @@ func (row grantRow) convert() *Grant {
 		GrantedBy:   NewAccountObjectIdentifier(row.GrantedBy),
 	}
 }
+
+// GrantOwnershipOptions is based on https://docs.snowflake.com/en/sql-reference/sql/grant-ownership#syntax.
+type GrantOwnershipOptions struct {
+	grantOwnership bool                    `ddl:"static" sql:"GRANT OWNERSHIP"`
+	On             OwnershipGrantOn        `ddl:"keyword" sql:"ON"`
+	To             OwnershipGrantTo        `ddl:"keyword" sql:"TO"`
+	CurrentGrants  *OwnershipCurrentGrants `ddl:"-"`
+}
+
+type OwnershipGrantOn struct {
+	SchemaObject GrantOnSchemaObject `ddl:"-"`
+}
+
+type OwnershipGrantTo struct {
+	// One of
+	DatabaseRoleName *DatabaseObjectIdentifier `ddl:"identifier" sql:"DATABASE ROLE"`
+	AccountRoleName  *AccountObjectIdentifier  `ddl:"identifier" sql:"ROLE"`
+}
+
+type OwnershipCurrentGrants struct {
+	RevokeOrCopy  OwnershipRevokeOrCopy `ddl:"-"`
+	currentGrants bool                  `ddl:"static" sql:"CURRENT GRANTS"`
+}
+
+type OwnershipRevokeOrCopy struct {
+	// One of
```

Comment:
this could be an enum called GrantOwnershipOutboundPrivileges with operations: REVOKE | COPY | NONE

Reply:
NONE is not a valid option, so I don't want to make it more complicated than necessary. I can change to enum with just two values.